src/sfctl/custom_version.py

```diff
@@ -0,0 +1,16 @@
+import pkg_resources
+from knack.util import CLIError
+
+def display_versions(client):
+    pkg = pkg_resources.get_distribution("sfctl")
+    sfctl_version = pkg.version
+    target_sf_version = None
+    for dependency in pkg.requires():
+        if dependency.key == 'azure-servicefabric':
+            # specs is a list of tuples for each version specified
+            target_sf_version = dependency.specs[0][1]
```

Comment:
I'm not sure this is really what you want, I think you want to identify the service fabric runtime that is compatible. There will always be lots of different versions of the `azure-servicefabric` package, `pip` is responsible for identifying the correct one, but we really want to know if I am using this version of `sfctl`, which version of the cluster do I want to connect to.

Reply:
So would there be a way to pull that information with it being hard coded or would that just be a constant somewhere?